### DISCUSSION TITLE
How do I enable stylistic sets for FiraCode in Godot Engine Code Editor?(And also, the text is not connected there, too)

### DISCUSSION THREAD

**Discussion by ghost** [tonsky/FiraCode#1195] [Q&A]
https://github.com/tonsky/FiraCode/wiki/How-to-enable-stylistic-sets
Here is the page that should tell me, and everyone else how to do it, but you know, nobody can do everything.
So, I decided to ask this question.
How do I enable them in Godot Engine?
This should surely be possible, right?

Also, the thing I heard is called "typographic ligatures" does not work there, for some reason.

**Comment by tonsky** [comment]
> This should surely be possible, right?

Not necessarily. Support depends on a program. For ligatures in code editors and terminals, there’s a compatibilty list in the README. For stylistic sets, support is even more scarce — almost nobody supports them fully.

If you figure out how to enable ligatures or ss in Godot, please report back, I’ll add them to the compatibilty list.

**Comment by lombervid** [comment]
Hello.

To enable font ligatures in Godot Engine's script editor (4.1.1).

In `Editor -> Editor Settings` menu, then in `Interface -> Editor` section:

- In `Code Font` select the font's file
- Then in `Code Font Contextual Ligatures` dropdown select:
  - `Enabled` to enable ligatures
  - `Use Custom OpenType Feature Set` to enable SS
    - In `Code Font Custom OpenType Features` place the SS codes separates by `,` without quotes or spaces: `ss01,ss02,ss03,ss04,ss05,ss06,zero,onum`

![image](https://github.com/tonsky/FiraCode/assets/3432651/b8e6e293-1cb6-47d6-b3ce-8bb5a894b059)
